addons/scoreboard/scoreboard.lua

```diff
@@ -75,6 +75,11 @@ windower.register_event('addon command', function()
             assert(loadstring(table.concat({...}, ' ')))()
             return
         end
+        
+        if command == nil then
+	    error('No command given. See //sb help')
+            return
+        end
 
         command = command:lower() or 'help'
```

Comment:
Should just display the help text instead of telling the user to use `//sb help`. Simplest way to do that is to change this line to this:
```lua
command = (command or 'help'):lower()
```
Then you don't need the nil check at all.